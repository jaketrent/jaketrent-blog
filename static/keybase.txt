==================================================================
https://keybase.io/jaketrent
--------------------------------------------------------------------

I hereby claim:

  * I am an admin of http://jaketrent.com
  * I am jaketrent (https://keybase.io/jaketrent) on keybase.
  * I have a public key with fingerprint DAD4 7EAA 5511 E975 3AE2  8F29 A7CA AAC8 CD78 9329

To claim this, I am signing this object:

{
    "body": {
        "key": {
            "fingerprint": "dad47eaa5511e9753ae28f29a7caaac8cd789329",
            "host": "keybase.io",
            "key_id": "a7caaac8cd789329",
            "uid": "a3aef91c1baeabd5157cab9022042d00",
            "username": "jaketrent"
        },
        "service": {
            "hostname": "jaketrent.com",
            "protocol": "http:"
        },
        "type": "web_service_binding",
        "version": 1
    },
    "ctime": 1405196545,
    "expire_in": 157680000,
    "prev": "d6c4935a7b51fc5eaeaeafa40ed45b7c67d5c276c72bd0a4f100eb9a4c4842b9",
    "seqno": 4,
    "tag": "signature"
}

with the aforementioned key, yielding the PGP signature:

-----BEGIN PGP MESSAGE-----
Version: Keybase OpenPGP v0.1.23
Comment: https://keybase.io/crypto

yMHkAnicbZFdSBRfGMZ3/fpYM7IULOrCI9GmsszMzuzsLKJYCUYXYlpkFMOZM2fX
0ZxZZ2f9wBQjREwC9UIRA6PawC7SyItUKjctgwQDMUSwG5U/Bonh4p/MxDNWN9G5
GeZ5f8/zvue8U8nRFpv11EZb53b+q1brh4nNoKU03GdtBJImNwBPI6jC+x+vovqw
7tcV1QAeIEOZ5TGEHEfTWOA5J8SM28sIkEcQQuRGMu8WnIwAckCFFjAdJEaCAexQ
NKKRH1GRifoPPvirQBK9Ao1oCWIoyRzNEVISKIahWEamKBMMYF2F1ZjQlbAKGzom
ozXlACLXKojIjfu9/0YcSKsmbr+uGRrSbpBShWH4PabTaPCbaB2WxN8hoqSoMrk5
MdRiPaBoKvDQhESGYqbSLMXRgotjuRyA6/2KjkXFJDje5abIMdvgWvO9XIgVnBzk
JY72Ig6TS2HohSyFZZaTeOTiZQ4xvAvxjCRTkPXSFIUlAbKIdbOMZL5LANeoGvCw
ZEzoI5EBxadCI6hj0PRm4lqMxWqzxMVGmbuz2BIP/dloeD5+l0590hwuj4uMX+xd
+HQ1rey/sfjkxZiuogELpSyB3G+zWUMzywMLZ/s/9uDI5NqLyfKZ8yur0e7idvtI
1OJRR1BNffmupKPwe96dr0vxj8szxecZ23ktbZEdx8JGfqTjeOvsreWa+6g3NNyc
tG2ctG9FdhNLm+yZhe0to6Gw1u1UUrp7U+5dGEubTdgS+wet6ZfOzAdHfOmDP1b9
J5JWpv+3hzLG688dPHz6QUHso6z160+fDQdHRVtx9rgdrBw50DI8vVMXa61bHll7
3Xdlp3NpyH/sZq6Y3NO1CTtv/1z8LF0eHHirTeUJc6H1hMqHRSV3s+feh2BB2Zf6
EuceWtEbCA==
=D/1C
-----END PGP MESSAGE-----

And finally, I am proving ownership of this host by posting or
appending to this document.

View my publicly-auditable identity here: https://keybase.io/jaketrent

==================================================================
