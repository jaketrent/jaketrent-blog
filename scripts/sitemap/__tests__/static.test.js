import { generateSitemap } from "../static"

describe("#generatedSitemap", () => {
  it("renders all static .tsx pages", async () => {
    const sitemap = await generateSitemap()
    expect(sitemap).toMatchSnapshot()
  })
})
