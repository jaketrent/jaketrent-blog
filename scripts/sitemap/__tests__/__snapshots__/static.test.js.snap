// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#generatedSitemap renders all static .tsx pages 1`] = `Promise {}`;
